### ISSUE TITLE
In-dApp notification and update ability

### ISSUE THREAD

**Issue by NutellaLicka** [TokelPlatform/tokel_dapp#217]
Tim requested something in the dApp that shows the users they need to update and also allows them to update from within the dApp.

I see these as two features:

Notification - The probably easier thing to implement

In-dApp updating - The definitely harder thing to implement

**Comment by piggydoughnut** [comment]
@acnebs I vaguely remember that you did work on some notifications of new versions when you were working on auto update? did you?

**Comment by acnebs** [comment]
Yep, I was toying with it, but quickly abandoned it since we had no automated build process and auto-updates weren't going to work anyway.

But now that we have the GH Workflows I'm happy to get at least a notification up and running at the very least. 👍

**Comment by piggydoughnut** [comment]
@acnebs yes that would be great, if people just got notified and got a link to where to get the new update from 🙏🏽  Thank you

**Comment by NutellaLicka** [comment]
Apologies, found this associated issue:
https://github.com/TokelPlatform/tokel_dapp/issues/64

Looks like we need signing auth to proceed with in-dapp updates.
